CMakeLists.txt

```diff
@@ -1233,6 +1252,9 @@ if(NOT ${universal_build})
     endif()
 endif()
 
+# setting ASTCENC_DECOMPRESSOR to ON breaks the build, so force ot to OFF
```

Comment:
s/ot/to/

Reply:
should've been "it" actually ("force *it* to off") - will fix, thanks for pointing it out :)